util/chplenv/chpl_libmode.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python
+import os
+import sys
+
+chplenv_dir = os.path.dirname(__file__)
+sys.path.insert(0, os.path.abspath(chplenv_dir))
+
+import overrides
+from utils import memoize
+
+
```

Comment:
I know you copied this from the hwloc version, but I'd drop down to 1 newline here (and update the hwloc one if you feel like it)